### ISSUE TITLE
bug: CLI launch bug config file not found

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1723]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.0.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.0.0...4.0.1) (2021-05-26)

- build(deps-dev): bump @types/express from 4.17.11 to 4.17.12 [`#1720`](https://github.com/open-wa/wa-automate-nodejs/pull/1720)
- Release 4.0.1 [`e13e60c`](https://github.com/open-wa/wa-automate-nodejs/commit/e13e60c1c2df59c3ec735739261a0069e88f3fd0)
- 🐛 fix: #1723 [`38ed67e`](https://github.com/open-wa/wa-automate-nodejs/commit/38ed67ee2d01a6e2fb0b688b236578398c0e2b19)
